test/unit-tests/type/unit/Unit.test.js

```diff
@@ -62,6 +62,16 @@ describe('Unit', function () {
       assert.strictEqual(unit1.units[0].unit.name, 'm3')
     })
 
+    it('should create a unit from an existing unit', function () {
+      const unit1 = new Unit(null, 'm/s^2')
+      const unit2 = new Unit(5, unit1)
```

Comment:
If the 'no strings in constructor' option is adopted, this would of course have to change to
`const unit1 = Unit.parse('m/s^2')`

Reply:
I think we can close this comment: for now we keep everything backward compatible so strings are still allowed in the constructor.